.mvn/maven.config

```diff
@@ -0,0 +1,2 @@
+-Pconsume-incrementals
+-Pmight-produce-incrementals
```

Comment:
The result of applying [this best practice](https://www.jenkins.io/doc/developer/tutorial-improve/enable-incrementals/).